status.go

```diff
@@ -39,7 +39,7 @@ const (
 	StatusRespBye StatusRespType = "BYE"
 )
 
-type StatusRespCode string
+type StatusRespCode RawString
```

Comment:
Hmm, `StatusRespCode` can't be used directly, since it's a new type (not `RawString`)

Reply:
Oh, right. I don't even remember why I changed it.